templates/server.conf.erb

```diff
@@ -408,8 +408,8 @@ enabled: <%= scope['pulp::email_enabled'] %>
 #   downloading content from the Squid cache.
 
 [lazy]
-enabled: <%= scope['pulp::lazy_enabled'] %>
-<% if scope['pulp::lazy_enabled'] -%>
+enabled: <%= scope['pulp::manage_squid'] %>
+<% if scope['pulp::manage_squid'] -%>
```

Comment:
you can delete these two lines (and the <% end -%> below it.  This option doesn't exist anymore (hence the removal of the puppet param for lazy_enabled)